klasses/bootcamp_admissions_client_test.py

```diff
@@ -103,59 +103,52 @@ def test_happy_path(test_data, mocked_get_200):
             "is_user_eligible_to_pay": True
         }
     ]
```

Comment:
`expected_payable_classes` is the same as `JSON_RESP_OBJ['bootcamps'][1]['klasses']`, could just use that instead here:

```python
expected_payable_klasses = JSON_RESP_OBJ['bootcamps'][1]['klasses']
```